lib/viewcomponents/graph.jsx

```diff
@@ -17,18 +17,14 @@ var Graph = React.createClass({
     },
 
     render: function() {
-        var data = this.props.model.getContents();
+        var graph = this.props.model.getGraph();
```

Comment:
If you replaced the only usage of the old method, then why did you only deprecate it, rather than delete it?